### PR TITLE
Add warning when interpolating to a boundary that contains multiple polylines

### LINKED ISSUE #370: Add warning when interpolating to a boundary that contains multiple polylines
A warning should be added when interpolating to a boundary that contains multiple polylines in one *.pli-file. See https://issuetracker.deltares.nl/browse/UNST-7012

### PR DESCRIPTION
**Pull request by Tammo-Zijlker-Deltares**
Fixes #370

### CODE REVIEW

File: tests/examples_workinprogress/workinprogress_modelbuilder.py [hunk 1]
```diff
@@ -135,6 +135,13 @@
 #TODO: two waterlevelbnds need to share same physical plifile in order to + (https://issuetracker.deltares.nl/browse/UNST-5320).
 list_quantities = ['salinitybnd','temperaturebnd','uxuyadvectionvelocitybnd','waterlevelbnd','tide'] #TODO: JV1 stable ("WARNING: Boundary link 00000468 already claimed")
 #list_quantities = ['waterlevelbnd','salinitybnd','temperaturebnd','uxuyadvectionvelocitybnd','tide'] #TODO: JV6 "ERROR  : update_ghostboundvals: not all ghost boundary flowlinks are being updated"
+
+#Check whether the polyfile contains multiple polyline, in that case show a warning
+pli = hcdfm.PolyFile(poly_file)
+if len(pli.objects) > 1:
+    print(f"Warning: the polyfile {poly_file} contains multiple polylines. Only the first one will be used by DFLOW-FM for the boundary conditions.")
```
Comment (veenstrajelmer) [inline tests/examples_workinprogress/workinprogress_modelbuilder.py:142]: This prints a string, consider whether an actual warning should be printed with `print(UserWarning(''))`. There are several built in warning/error types available: https://docs.python.org/3/library/warnings.html#warning-categories. Sometimes there is a need for an extra warning/error class like so:
https://github.com/Deltares/dfm_tools/blob/62ed9b8fdbc918ded534ee4eb933dbeee6fe0968/dfm_tools/errors.py#L8C1-L9
